.github/ISSUE_TEMPLATE/Cloud_service.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 🛠 New Cloud Service Request
+about: To create a request for a new set of cloud service controls
+
+---
+
+## New Cloud Service
+
+A new cloud service must follow the standard completion template. The 'definition of done' of a cloud service request is defined as follows:
+
+- [ ] Cloud Platform Target
+- [ ] Service Name
+- [ ] Service Approval Accelerator Definition
+- [ ] Control Framework Definition
+- [ ] Test Cases Created
```

Comment:
Merge the Control Framework and test cases step together to mirror what we have for redshift

Reply:
+1

Reply:
Thanks for the +1 @abdullahgarcia 👍 

@fleadsom - Please let @abdullahgarcia know when the changes have been done so the PR can be merged into master.

Cheers,

James.